.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '18' ]
+        node: [ '16', '18' ]
```

Comment:
question: Are you sure you would like to have a node version of 16 here? It is deprecated AFAIK

Reply:
im not opposed to not having it - in my mind i didn't want to prevent older systems from not having support

@Baroshem @matiasfha do you have minimum node versions on your respective libraries?

given its going to be released as breaking change, its a good opportunity to pin a minimum version

Reply:
I do have a 18.X in most of my project which I think could be safer :)